foundation/foundation-mda/src/main/resources/templates/data-delivery-pyspark/abstract.data.action.py.vm

```diff
@@ -51,7 +51,7 @@ class AbstractDataAction(ABC):
         builder = SparkSession.builder
         if properties.getProperty("execution.mode.legacy", "false") == "true":
             builder = builder\
-                .master("local[*]")\
+                .master("local[*]") \
```

Comment:
A: Unrelated to this ticket but we really need to just remove this concept of legacy execution mode.